releasenotes/notes/add_forwarder_storage_max_size_in_bytes-d860cf200534df6b.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
```

Comment:
```suggestion
# Each section from every release note are combined when the
```

Reply:
I fixed the typo in the template. See https://github.com/DataDog/datadog-agent/pull/6940.